elixir/apps/web/lib/web/controllers/error_controller.ex

```diff
@@ -4,7 +4,6 @@ defmodule Web.ErrorController do
   def show(_conn, params) do
     case params["code"] do
       "404" -> raise Web.LiveErrors.NotFoundError
-      "422" -> raise Web.LiveErrors.InvalidRequestError
```

Comment:
Just curious why this (and the test for 422) is being removed?

Reply:
We don't have that exception at all now

Reply:
🤔 I could have sworn I ran in to that when I created those error pages, which is why I put it in.  Let me see if I can reproduce what I saw.

Reply:
@bmanifold this PR removes it :)

Reply:
Done